software/base/src/test/java/org/apache/brooklyn/entity/software/base/VanillaSoftwareProcessStreamsIntegrationTest.java

```diff
@@ -59,8 +39,24 @@ public void setUp() throws Exception {
     }
 
     @Test(groups = "Integration")
+    @Override
     public void testGetsStreams() {
-        Map<String, String> cmds = ImmutableMap.<String, String>builder()
```

Comment:
Slightly simpler to do `Map<String, String> cmds = getCommands();`. And then below to use `cmds` rather than repeatedly calling `getCommands()`.